lib/snmplib/mib.c

```diff
@@ -274,11 +268,7 @@ int objidlen;       /* number of subidentifiers */
 
 }
 
-void
-sprint_objid(buf, objid, objidlen)
-char *buf;
-oid *objid;
-int objidlen;       /* number of subidentifiers */
+void sprint_objid(char *buf, oid *objid, int objidlen)
```

Comment:
Restoring formatting and lost comment:

```suggestion
/// \param objidlen number of subidentifiers
void
sprint_objid(char *buf, oid *objid, int objidlen)
```